schemas/org.gnome.shell.extensions.gtile.gschema.xml

```diff
@@ -173,49 +213,89 @@
 		</key>   
 		
 		<key type = "s" name="resize11">
-		       <default>'3x3 0:0 0:0'</default>
+		       <default>'2x3 0:2 0:2'</default>
   	  	       <summary>Preset resize11 settings.</summary>
 		</key>   
 		<key type = "s" name="resize12">
-		       <default>'3x3 0:1 0:1'</default>
+		       <default>'3x3 0:2 1:2'</default>
```

Comment:
I think this one should be ```2x3 0:2 1:2```. By the description on help: ```2 - lower third```. Currently if you combine this one with ```Super+Ctrl+8``` and ```Super+Ctrl+5```, you leave the bottom right tile empty.